docs/README.md

```
@@ -0,0 +1,42 @@
+* Python 3.10 or higher
+* [Task] 3.40.0 or higher
+
+## Build commands
+
+* Build the site incrementally:
+
+  ```shell
+  task docs:site
+  ```
+
+    * The output of the build will be in `../build/docs/html`.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)